docs/source/cli_help/clean.rst

```diff
@@ -0,0 +1,25 @@
+.. _clean:
```

Comment:
Whoops this piece of docs isn't referenced/included ‼️ 

![image](https://user-images.githubusercontent.com/3383210/156378456-5eae64a5-5be5-4d5f-bd45-5d4cc49a2d15.png)